schema/annotation-source.yaml

```diff
@@ -161,30 +151,51 @@ $defs:
       A Statement (aka ‘Assertion’) represents a claim of purported truth as made by a particular agent, 
       on a particular occasion.
     heritable_properties:
+      target_proposition:
+        oneOf:
+        - $ref: "#/$defs/Proposition"
+        - $ref_curie: gks.core:CURIE
+        description: The Proposition about which the Statement is made.
+      evidence_direction:
+         type: string
+         enum: [ "supports", "disputes", inconclusive" ] 
+         description: >-
+           A term indicating the net direction of all evidence assessed against the target proposition. 
       evidence_level:
         $ref_curie: gks.core:Coding
         description: >- 
-          A term indicating the overall strength of support for the Statement based on all evidence assessed.
+          A term indicating the overall strength of support for or against the Statement's proposition, 
+          as provided by all evidence evaluated.
       evidence_score:
         $ref: "#/$defs/DataItem"
         description: >-
-          A quantitative score reflecting the overall strength of support for the Statement based on all 
-          evidence assessed.
-      target_proposition:
-        oneOf:
-        - $ref: "#/$defs/Proposition"
-        - $ref_curie: gks.core:CURIE
-        description: The Proposition about which the Statement is made.
-      conclusion:
+          A quantitative score reflecting the overall strength of support for or against the Statement's 
+          proposition, as provided by all evidence evaluated.
+      confidence_level: 
         $ref_curie: gks.core:Coding
         description: >-
-          The conclusion drawn from the statement proposition, direction, strength, and/or 
-          confidence score.
-      direction:
+          A term indicating the asserting agent's level of confidence that the target 
+          proposition is true or false (the direction this assessment pertains to is captured
+          by the confidence_direction field).
+      confidence_score: 
+        $ref: "#/$defs/DataItem" 
+        description: >-
+          A quantitative score term reflecting the asserting agent's level of confidence that
+          the target proposition is true or false (the direction this assessment pertains to 
+          is captured by the confidence_direction attribute).
+      confidence_direction: 
         type: string
-        enum: [ "supports", "uncertain", "opposes" ]
+        enum: [ "true", "false", "uncertain" ]
```

Comment:
> "true", "false"

IMO these direction terms make the most sense when talking about confidence direction - as we aim to distinguish whether the reported level reflects confidence that the proposition is "true", vs "false". 

> "uncertain"

The term "uncertain" would be used when the curator is not certain about the direction of their confidence (and in this case, confidence_level/score is not applicable)